S21/myadlapalli/Language.py

```diff
@@ -0,0 +1,72 @@
+NO_CHANGE_LANGS = ['zh', 'ja', 'th']
+TURKISH_LANGS = ['tr', 'az']
+
+IRISH = 'ga'
+IRISH_VOWELS = ['A', 'E', 'I', 'O', 'U', 'Á', 'É', 'Í', 'Ó', 'Ú']
+
+GREEK = 'el'
+
+
+def toLower(sentence, lang):
+    sentence = sentence.strip()
+    result = ''
+    lang = lang.split('-')[0]
+
+    # Languages with no change
+    if lang in NO_CHANGE_LANGS:
+        return sentence
+
+    words = sentence.split(" ")
```

Comment:
We can follow the google style guide for better code writing principles by not adding blank lines between the code except function or class declaration.